stock_location_analytic_purchase/README.rst

```diff
@@ -0,0 +1,28 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Procurement Location Analytic
```

Comment:
This is not your fault, but can you change this title please? It is wrong. Better "Purchasing for analytic locations".